### PR TITLE
Added new ValidatorOptions validation rule and improved readability of existing error + tests.

### LINKED ISSUE #1774: Validator keys option should give better error message
When setting the following options:

- validators-key-file
- validators-key-password-files

I get a generic error message which doesn't tell me what the actual problem is. It would be good to know if the file cannot be found (wrong path), data is invalid (correct path but wrong keyfile or password) or something else.

Current response is:
`Invalid configuration. The size of validator.validatorsKeystoreFiles [0] and validator.validatorsKeystorePasswordFiles [1] must match`

### PR DESCRIPTION
**Pull request by mark-terry**
## PR Description
Added some extra validation for ValidatorOptions CLI options and improved readability of the existing message.  New tests added.

## Fixed Issue(s)

Fixes #1774

### CODE REVIEW

File: util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java [hunk 1]
```diff
@@ -456,17 +457,30 @@ public void validateConfig() throws IllegalArgumentException {
...
 
-  private void validateKeyStoreFilesAndPasswordFilesSize() {
+  private void validateKeyStoreFilesAndPasswordFilesConfig() {
     final List<String> validatorKeystoreFiles = getValidatorKeystoreFiles();
     final List<String> validatorKeystorePasswordFiles = getValidatorKeystorePasswordFiles();
 
+    if ((validatorKeystoreFiles != null && validatorKeystorePasswordFiles == null)
+        || (validatorKeystoreFiles == null && validatorKeystorePasswordFiles != null)) {
+      final String errorMessage =
+          "Invalid configuration. '--validators-key-files' and '--validators-key-password-files' cannot be specified separately";
```
Comment (macfarla) [inline util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java]:
```suggestion
          "Invalid configuration. '--validators-key-files' and '--validators-key-password-files' must be specified together";
```

File: util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java [hunk 2]
```diff
@@ -456,17 +457,30 @@ public void validateConfig() throws IllegalArgumentException {
...
+      StatusLogger.getLogger()
+          .debug(
+              "Invalid configuration. The size of validator.validatorsKeystoreFiles {} and validator.validatorsKeystorePasswordFiles {} must match",
+              validatorKeystoreFiles.size(),
+              validatorKeystorePasswordFiles.size());
+
       final String errorMessage =
           String.format(
-              "Invalid configuration. The size of validator.validatorsKeystoreFiles [%d] and validator.validatorsKeystorePasswordFiles [%d] must match",
+              "Invalid configuration. The number of arguments for --validators-key-files [%d] and --validators-key-password-files [%d] must match",
```
Comment (macfarla) [inline util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java]: number of arguments doesn't seem quite right. The argument is a list, containing a number of items? Size or length might be more accurate

Comment (mark-terry) [reply util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java]: We're trying to make it more human-readable, imo referring to the number of items in a list as size or length is confusing.

Perhaps something more like: 
`"The number of files specified for --validators-key-files [%d] and --validators-key-password-files [%d] must match"` ?

Comment (rolfyone) [reply util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java]: `each key file specified by --validators-keys-files must also have a password file specified by --validators-key-password-files`?

Comment (macfarla) [reply util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java]: you must specify the same number of key and password files?

Comment (mark-terry) [reply util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java]: The number of --validators-key-files (num) must equal the number of --validators-key-password-files (num)

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**